_episodes/001-ruma.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+title: "Ruma and the Matrix Communication Protocol: An Interview with Jimmy Cuadra"
+date: 2019-08-08T11:45:00+0000
+file: https://audio.rustacean-station.org/file/rustacean-station/rustacean-station-e001-ruma-matrix-jimmy-cuadra.mp3
+duration: "1:03:29" #audio length in min
+length: "60946851" #filesize in byte
+
+# https://github.com/jekyll/jekyll/issues/7744
+layout: episode
+---
+
+We interview Jimmy Cuadra about Matrix, an open and decentralized communication protocol, and his implementation in Rust known as [Ruma](https://github.com/ruma/ruma).
+
+ - Twitter: [@rustaceanfm](https://twitter.com/rustaceanfm)
+ - Discord: [Rustacean Station](https://discord.gg/cHc3Gyc)
+ - Github: [@rustacean-station](https://github.com/rustacean-station/)
+ - Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+### Timestamps & referenced resources
+
+* 0:01:35 - Meet Jimmy Cuadra
```

Comment:
At least pocket casts parses these time stamps as links when written like `[@01:35]`

Reply:
Interesting, I'll try that.